task/udprpc/src/main.rs

```diff
@@ -141,12 +141,28 @@ fn main() -> ! {
                     RpcReply::Ok => (nreply + REPLY_PREFIX_SIZE) as u32,
                 };
 
-                net.send_packet(
-                    SOCKET,
-                    meta,
-                    &tx_data_buf[0..(meta.size as usize)],
-                )
-                .unwrap();
+                loop {
+                    match net.send_packet(
+                        SOCKET,
+                        meta,
+                        &tx_data_buf[0..(meta.size as usize)],
+                    ) {
+                        Ok(()) => break,
+                        // If our tx queue is full or `net` just restarted, just
+                        // retry our send; both of these should be ephemeral.
+                        Err(
+                            SendError::QueueFull | SendError::ServerRestarted,
+                        ) => continue,
```

Comment:
This behavior is probably not right; it will just burn CPU on QueueFull. It would be more polite to wait for notification from the netstack that space is available.